src/include/sof/audio/component_ext.h

```diff
@@ -147,13 +148,46 @@ static inline int comp_trigger_remote(struct comp_dev *dev, int cmd)
 	return idc_send_msg(&msg, IDC_BLOCKING);
 }
 
+static inline int comp_trigger_local(struct comp_dev *dev, int cmd)
+{
+	int ret;
+
+	ret = dev->drv->ops.trigger(dev, cmd);
```

Comment:
Dont we need to use the wrapper API here ?

Reply:
> hat the next LL will just prempt the DP task and return to it again after the next LL

Yes, DP tasks (not a single task, multiple tasks) are to be run as threads, lower priority than LL. 

Of course if a DP module is triggered less frequently than LL, the buffers between them must be big enough

Reply:
> Dont we need to use the wrapper API here ?

you mean **static inline int comp_trigger** ?
No, comp_trigger calls comp_trigger_local or comp_trigger_remote, so we're in api wrapper in fact